share/github-backup-utils/ghe-backup-settings

```diff
@@ -26,16 +26,51 @@ echo "* Transferring license data ..." 1>&3
 ghe-ssh "$host" -- "sudo cat '$GHE_REMOTE_LICENSE_FILE'" > enterprise.ghl
 
 # Function to backup a secret setting to a file.
-#   backup-secret <description> <file-name> <setting-name>
+#   backup-secret <description> <file-name> <setting-name> 
 backup-secret() {
-  echo "* Transferring $1 ..." 1>&3
-  ghe-ssh "$host" -- ghe-config "$3" > "$2+" || (
-    echo "Warning: $1 not set" >&2
+
+  best_effort=false
+  label=""
```

Comment:
Let's name the `label` vs `description` (in comment above) consistently 😁

Reply:
optionalI applied this, but I fat-finged the "close" button so I had to do it manually 😄